backends/advanced/src/advanced_omi_backend/routers/modules/obsidian_routes.py

```
@@ -0,0 +1,200 @@
+):
+    """
+    Immediate/synchronous ingestion endpoint (legacy). Not recommended for UI.
+    Prefer the upload_zip + start endpoints to enable progress reporting.
+    """
+    if not os.path.exists(request.vault_path):
+        raise HTTPException(status_code=400, detail=f"Path not found: {request.vault_path}")
+
+    try:
+        result = obsidian_service.ingest_vault(request.vault_path)
+        return {"message": "Ingestion complete", **result}
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)